plugins/generic/acron/AcronPlugin.inc.php

```diff
@@ -244,7 +244,11 @@ function shutdownFunction() {
 
 			// By updating the last run time as soon as feasible, we can minimize
 			// the race window. See bug #8737.
-			$updateResult = $taskDao->updateLastRunTime($className, time());
+			$tasksToRun = $this->_getTasksToRun();
```

Comment:
I mean should there be something like `$this->_tasksToRun = $tasksToRun` after this loop? -- because `$tasksToRun` might be different than `$this->_tasksToRun` at this point. I actually don't think so, but... wanted to ask however... :-)